### ISSUE TITLE
RateLimitProcessor

### ISSUE THREAD

**Issue by bossenti** [apache/streampipes#1894]

**Comment by PalaniappanC** [comment]
Hi @bossenti @dominikriemer @tenthe @RobertIndie I Would like to work on this issue. Can you assign this to me

**Comment by tenthe** [comment]
Hi @ I'm glad you want to work on the issue. I have assigned you. Please let us know if you need any help.

**Comment by rahulbiswas876** [comment]
@tenthe may I take up the task

**Comment by bossenti** [comment]
@PalaniappanC are you still interested to contribute here?

@rahulbiswas876 feel free to take up one of the [unassigned issues](https://github.com/apache/streampipes/issues?q=is%3Aissue+is%3Aopen+no%3Aassignee)
If you want to be assigned feel free to leave a comment there, so we can assign you

**Comment by PalaniappanC** [comment]
@bossenti Yup I have looked at the documentation and understood how rate limit processor works. Will work on this next weekend as I am travelling to lot of places this week

**Comment by bossenti** [comment]
Great to hear!
No need for a hurry, just wanted to know if this issue is stale 🙂

**Comment by bossenti** [comment]
In case you need any kickstart, feel free to approach us
